Bottom: 2a55fbe765ae79e9ea6a0b34aa42b826e9a61ccb
Top:    40a3cfe4a63875a05ff2df5deec398f3c2874096
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:30:46 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 3362da6..6fa8575 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -324,13 +324,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
